.github/workflows/source_build.yml

```diff
@@ -15,6 +19,36 @@ jobs:
         run: |
           pip install flake8
           flake8 .
+      - name: install dependencies
+        run: |
+          sudo apt-get install -y git
+      - name: Build Bio-Formats
+        id: bf
+        if: matrix.build_bf
+        run: |
+          git clone https://github.com/ome/bioformats /tmp/bioformats --depth 1
+          cd /tmp/bioformats
+          mvn install -DskipTests -q
+          echo "::set-output name=bf_version::$(mvn help:evaluate -Dexpression=project.version -q -DforceStdout)"
+          echo "::set-output name=ome_common_version::$(mvn help:evaluate -Dexpression=ome-common.version -q -DforceStdout)"
+      - name: Build ZarrReader
+        id: zarr_reader
+        if: matrix.build_zarr
+        run: |
+          git clone -b dependency-updates https://github.com/dgault/ZarrReader /tmp/ZarrReader
+          cd /tmp/ZarrReader
+          mvn install -DskipTests -q
+          echo "::set-output name=zarr_reader_version::$(mvn help:evaluate -Dexpression=project.version -q -DforceStdout)"
+      - name: Set ZarrReader version
+        if: matrix.build_zarr
+        run: |
+          sed -i 's/versions.OMEZarrReader=.*/versions.OMEZarrReader=${{ steps.zarr_reader.outputs.zarr_reader_version }}/' etc/omero.properties
```

Comment:
If there is an error in the substitution e.g. a variable name change, will the `sed` operation fail or leniently pass?

An alternative would be to write versions to `etc/local.properties` which should override `etc/omero.properties`.

Reply:
``sed`` will not fail as such but the build will afterwards

Reply:
usage of ``etc/local.properties`` is an option but not sure it will give us much in that case. If the variable name is changed for example, the adjustment will have to be made to in the workflow in order to be taken into account